build.sbt

```diff
@@ -57,22 +55,6 @@ lazy val microservice = Project(appName, file("."))
         }.value)
     )
   )
```

Comment:
you will need to create a service manager profile for these - such as "DC_SECURE_MESSAGE_IT".
Then add an entry into build-jobs for the SbtMicroserviceJobBuilder(TEAM, 'secure-message'), replacing .withSmServer with .withServiceManager2("DC_SECURE_MESSAGE_IT").

Reply:
@ardwalker thats already been done and merged it as well.. thanks

Reply:
Great, you might want to add a quick note to the readme describing how to run the integration tests - because now you need to run `sm2 --start DC_SECURE_MESSAGE_IT` before and same but with --stop after you've run them.